storage/response_cache.go

```diff
@@ -101,9 +99,6 @@ func (rc *ResponseCache) GetResponse(e engine.Engine, cmdID proto.ClientCmdID, r
 // return an error. The copy is done directly using the engine instead
```

Comment:
Sure, removed the sentence starting from "The copy is done ...". Also removed "The cache will be locked while copying is in progress".